test/hotspot/jtreg/compiler/c2/TestMergeLoads.java

```diff
@@ -0,0 +1,706 @@
+        testGroups.get("test2").put("test2c", (_,_) -> { return test2c(aB.clone()); });
+
+        // Get int in big endian
+        testGroups.put("test3", new HashMap<String,TestFunction>());
+        testGroups.get("test3").put("test3R", (_,_) -> { return test3R(aB.clone()); });
+        testGroups.get("test3").put("test3a", (_,_) -> { return test3a(aB.clone()); });
+        testGroups.get("test3").put("test3b", (_,_) -> { return test3b(aB.clone()); });
+        testGroups.get("test3").put("test3c", (_,_) -> { return test3c(aB.clone()); });
+
+        // Get long in big endian
+        testGroups.put("test4", new HashMap<String,TestFunction>());
+        testGroups.get("test4").put("test4R", (_,_) -> { return test4R(aB.clone()); });
+        testGroups.get("test4").put("test4a", (_,_) -> { return test4a(aB.clone()); });
+        testGroups.get("test4").put("test4b", (_,_) -> { return test4b(aB.clone()); });
+        testGroups.get("test4").put("test4c", (_,_) -> { return test4c(aB.clone()); });
+
+        // Merge char as int
+        testGroups.put("test5", new HashMap<String,TestFunction>());
+        testGroups.get("test5").put("test5R", (_,_) -> { return test5R(aC.clone()); });
+        testGroups.get("test5").put("test5a", (_,_) -> { return test5a(aC.clone()); });
+        testGroups.get("test5").put("test5b", (_,_) -> { return test5b(aC.clone()); });
+
+        // Merge char as long
+        testGroups.put("test6", new HashMap<String,TestFunction>());
+        testGroups.get("test6").put("test6R", (_,_) -> { return test6R(aC.clone()); });
+        testGroups.get("test6").put("test6a", (_,_) -> { return test6a(aC.clone()); });
+        testGroups.get("test6").put("test6b", (_,_) -> { return test6b(aC.clone()); });
+
+        // Merge short as int
+        testGroups.put("test7", new HashMap<String,TestFunction>());
+        testGroups.get("test7").put("test7R", (_,_) -> { return test7R(aS.clone()); });
+        testGroups.get("test7").put("test7a", (_,_) -> { return test7a(aS.clone()); });
+        testGroups.get("test7").put("test7b", (_,_) -> { return test7b(aS.clone()); });
+
+        // Merge short as long
+        testGroups.put("test8", new HashMap<String,TestFunction>());
+        testGroups.get("test8").put("test8R", (_,_) -> { return test8R(aS.clone()); });
+        testGroups.get("test8").put("test8a", (_,_) -> { return test8a(aS.clone()); });
+        testGroups.get("test8").put("test8b", (_,_) -> { return test8b(aS.clone()); });
+    }
+
+    static void set_random(byte[] a) {
+        for (int i = 0; i < a.length; i++) {
+            a[i] = (byte)RANDOM.nextInt();
+        }
+    }
+
+    static void set_random(char[] a) {
+        for (int i = 0; i < a.length; i++) {
+            a[i] = (char)RANDOM.nextInt();
+        }
+    }
+
+    static void set_random(short[] a) {
+        for (int i = 0; i < a.length; i++) {
+            a[i] = (short)RANDOM.nextInt();
+        }
+    }
```

Comment:
```suggestion
    }

```
Please put a space between methods ;)

Reply:
Ok